code/game/machinery/cell_charger.dm

```diff
@@ -124,10 +124,16 @@
 
 /obj/machinery/cell_charger/process(delta_time)
 	if(!charging || !anchored || (machine_stat & (BROKEN|NOPOWER)))
+		//to_chat(world, "DEBUG -- CELL CHARGER EARLY RETURN(generic error)")
 		return
 
 	if(charging.percent() >= 100)
 		return
-	if(directly_use_power(charge_rate * delta_time))
-		charging.give(charge_rate * delta_time)
-		update_icon()
+
+	var/main_draw = use_power_from_net(charge_rate * delta_time, take_any = TRUE) //Pulls directly from the power net for the cell itself
```

Comment:
```suggestion
	var/main_draw = use_power_from_net(charge_rate * delta_time, take_any = TRUE) //Pulls directly from the PowerNet for the cell itself
```

Reply:
its not even consistent how you change the word power net